### PR TITLE
deps: upgrade dependencies

### PR DESCRIPTION
**Pull request by benjycui**
ref: https://github.com/ant-design/ant-design/issues/1538

### CODE REVIEW

**Comment by benjycui** [comment]
Done.

**Comment by benjycui** [comment]
Rebased.

File: components/tabs/style/index.less [hunk 1]
```diff
@@ -159,30 +156,26 @@
...
-        .@{iconfont-css-prefix} {
-          width: 14px;
-          height: 14px;
-          margin-right: 8px;
-        }
+      &:active {
+        color: shade(@primary-color, 5%);
+      }
+      .@{iconfont-css-prefix} {
+        width: 14px;
```
Comment (afc163) [inline components/tabs/style/index.less]: 缩进少了。

File: components/tabs/style/index.less [hunk 2]
```diff
@@ -394,14 +323,14 @@
...
-  &&-card > &-bar &-tab-active &-tab-inner {
+  &&-card > &-bar &-tab-active {
     padding-bottom: 7px;
     transform: translateZ(0);
   }
   &&-card > &-bar &-nav-wrap {
     margin-bottom: 0;
   }
-  &&-card > &-bar &-tab-inner .@{iconfont-css-prefix}-cross {
+  &&-card > &-bar &-tab .@{iconfont-css-prefix}-cross {
```
Comment (afc163) [inline components/tabs/style/index.less]: #3041 之后，改成 close

**Pull request merged**